travis-mono-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+set -x
+
+for dir in MoreLinq.Test/bin/$CONFIGURATION/net451/*/ ; do
+	mono $dir/dotnet-test-nunit.exe $dir/MoreLinq.Test.dll
```

Comment:
This adds a double slash here, as in:

    mono MoreLinq.Test/bin/Release/net451/ubuntu.14.04-x64//dotnet-test-nunit.exe \
         MoreLinq.Test/bin/Release/net451/ubuntu.14.04-x64//MoreLinq.Test.dll

Threw me off when searching through the log. Would it be nitpicking to trim the trailing slash using `{$dir%/}`, like below?

    mono ${dir%/}/dotnet-test-nunit.exe ${dir%/}/MoreLinq.Test.dll

And would it be even more nitpicking to replace the leading tab on that line with spaces instead?